backend/multi-account/backups/app.py

```
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+from aws_cdk import App, Environment
+from stacks.backup_account_stack import BackupAccountStack
+
+
+class BackupsApp(App):
+    def __init__(self, *args, **kwargs):
+        """
+        An app for deploying data retention backup destination infrastructure in the backup account.
+
+        This application creates the centralized backup infrastructure that receives and stores
+        backup copies from CompactConnect environment accounts:
+        - Cross-account backup vaults (general and SSN-specific)
+        - Customer-managed KMS keys for backup encryption
+        - Organization-level access policies for secure cross-account backup operations
+
+        Environment account backup infrastructure (local vaults, IAM roles, backup plans)
+        is managed by the CompactConnect application deployment.
+        """
```

If I recall correctly, it was mentioned that if these vaults don't exist in the secondary backup account for whatever reason, the application pipeline deployments will still succeed, but the backup processes will silently fail. If this is true, is there a way we can add alerting for this in prod? We will need to know if the backup processes are failing for whatever reason.

---

We've confirmed through testing that if the copy job fails to copy for whatever reason, it currently sends out a slack notification. However, if the vault and backup plans are deleted, there are no alerts sent out since no copy jobs are performed after that point. This should be acceptable for MVP, though we may want to consider preventing even admins from deleting these vaults.

---

I was able to clear out our entire 'prod' testing environment within a matter of 15 minutes, so if a bad actor gained access to our account the backups could be wiped out fairly quickly, though I recall @jusdino mentioning that we can create a lock on these vaults that essentially prevents anyone from deleting them. I'm not sure if the intent was to set that lock configuration as part of this PR now, or if that was a future effort.